images/linux/scripts/installers/android.sh

```diff
@@ -73,6 +73,8 @@ availableBuildTools=$(echo ${allBuildTools[@]//*rc[0-9]/})
 filter_components_by_version $minimumPlatformVersion "${availablePlatforms[@]}"
 filter_components_by_version $minimumBuildToolVersion "${availableBuildTools[@]}"
 
+components+=" cmdline-tools;latest"
```

Comment:
Maybe add it to the toolset?

Reply:
well, no. It was done from the very first commit, but after discussion we decided to do not pollute the image with the many obviously useless old versions and install only the latest one. As a result there's no need to have an extra setting in the toolset.json.  If i understood you right.

Reply:
I think we can add ` cmdline-tools;latest` to `additional_tools` section: https://github.com/actions/virtual-environments/blob/main/images/linux/toolsets/toolset-1804.json#L87

Reply:
it should work for some other images too